modules/combined/test/tests/contact_verification/patch_tests/plane_3/plane3_template1.i

```diff
@@ -263,8 +263,8 @@
   type = Transient
   solve_type = 'PJFNK'
 
-  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
-  petsc_options_value = 'lu       superlu_dist'
+  petsc_options_iname = '-pc_type'
+  petsc_options_value = 'asm'
```

Comment:
Does LU throw out errors on this?

Reply:
It did on linux but ran fine on Mac. I would like to re-run this on rod without superlu_dist

Reply:
Yea, so I just ran this on cone, and with `superlu_dist` we get this solve behavior:
```
 0 Nonlinear |R| = 7.770396e+01
Solution, reference convergence variable norms:
disp_x: 0  saved_x: 0
disp_y: 77.704  saved_y: 0
0       Linear |R| = 7.770396e+01
1       Linear |R| = 7.770171e+01
2       Linear |R| = 1.062272e+01
3       Linear |R| = 2.589127e+00
4       Linear |R| = 1.132553e+00
5       Linear |R| = 2.354931e-01
6       Linear |R| = 1.057685e-01
7       Linear |R| = 4.257440e-02
1  Nonlinear |R| = 3.262283e+03
```
and the solve eventually fails with the message: `Failure in NearestNodeThread because solution contains inf or not-a-number entries.  This is likely due to a failed factorization of the Jacobian matrix.`

If I no longer use superlu_dist (instead using petsc's lu), the the solve succeeds easily:
```
 0 Nonlinear |R| = 7.770396e+01
Solution, reference convergence variable norms:
disp_x: 0  saved_x: 0
disp_y: 77.704  saved_y: 0
0       Linear |R| = 7.770396e+01
1       Linear |R| = 2.253578e+01
2       Linear |R| = 2.626581e-11
1  Nonlinear |R| = 3.330550e-03
```
So it looks like there's still some buggy behavior in `superlu_dist` as @bwspenc has experienced before